app/uk/gov/hmrc/essttpstubs/controllers/EligibilityController.scala

```diff
@@ -45,6 +45,7 @@ class EligibilityController @Inject() (
   }
 
   def retrieveEligibilityData: Action[EligibilityRequest] = Action.async(parse.json[EligibilityRequest]) { implicit request =>
+    logger.debug(s"EligibilityRequest: ${Json.prettyPrint(Json.toJson(request.body))}")
```

Comment:
since this is our stub we could probably get away with doing this at info level and maybe log the request body and response status and body in the same log. Something like `"responding to ${request.uri} request with request body ... with response status ... and response body ..."` maybe?

Reply:
I've updated, but I have done it in separate log statements. This is only because the response bodies can get quite large (number of lines of json) and I worry it may be harder for us to decipher if we include request body and response body in the same log, hope that's ok

Reply:
👍 that makes sense